pkg/parse/quote_test.go

```diff
@@ -24,8 +24,8 @@ func TestQuote(t *testing.T) {
 
 		// Double quote when there is unprintable char.
 		Args("a\nb").Rets(`"a\nb"`),
-		Args("\x1b\"\\").Rets(`"\e\"\\"`),
-		Args("\x00").Rets(`"\x00"`),
+		Args("\000\x1b\"\\").Rets(`"\x00\e\"\\"`),
+		Args("\u0000").Rets(`"\x00"`),
```

Comment:
The change from `\x00` to `\u0000` seems unnecessary; these two Go literals are equivalent, and the change doesn't have any effect on which Elvish code is hit.

Reply:
Yes, per my reply to to your previous observation of this same point this is simply testing that Go translates both sequences to a NUL byte rather than testing that Elvish translates both sequences to a NUL byte. TBD is how to do the latter. I've eliminated the redundant test.